### PR TITLE
SISRP-31636 - Hard-codes the committee name

### PR DESCRIPTION
**Pull request by lyttam**
https://jira.berkeley.edu/browse/SISRP-31636

The field we were relying upon for the committee name is being repurposed for eForm-related data.  CalCentral needs to be smart enough to map the committee type code to a user-friendly name.

### CODE REVIEW

File: app/models/my_committees/committees_module.rb [hunk 1]
```diff
@@ -34,9 +47,17 @@ def parse_cs_committee (cs_committee)
     committee
   end
 
+  def translate_committee_type(cs_committee)
+    COMMITTEE_TYPES[cs_committee[:committeeType].try(:intern)].try(:[], :label)
+  end
+
+  def qualifying_exam?(cs_committee)
+    cs_committee[:committeeType].to_s.strip.upcase === COMMITTEE_TYPES[:QE][:code]
```
Comment (nkutub) [inline app/models/my_committees/committees_module.rb]: @lyttam , did you forget the try here? on line 51 you used a try for the same attribute  :committeeType

Comment (lyttam) [inline app/models/my_committees/committees_module.rb]: @nkutub the `try` on line 51 is for `intern` and `label`.  On lines 51 and 55, I'm assuming the data from CS will include the COMMITTEE_TYPE element.  Do you think I should add a `try` for that as well?  I would hope that the API contract ensures each data element will be present in the response, even if it's null.

Comment (nkutub) [inline app/models/my_committees/committees_module.rb]: @lyttam , sorry for some reason I thought that nil.to_s would error but it does not. It looks good to me

File: app/models/my_committees/committees_module.rb [hunk 2]
```diff
@@ -34,9 +47,17 @@ def parse_cs_committee (cs_committee)
     committee
   end
 
+  def translate_committee_type(cs_committee)
+    COMMITTEE_TYPES[cs_committee[:committeeType].try(:intern)].try(:[], :label)
```
Comment (nkutub) [inline app/models/my_committees/committees_module.rb]: @lyttam, ok I don't understand this line :(. Also, I think you are missing ] at end of line. I don't see sample data that shows that you will have { committeeType: { intern: { label: 'value' } } }

File: app/models/my_committees/committees_module.rb [hunk 3]
```diff
@@ -34,9 +47,18 @@ def parse_cs_committee (cs_committee)
     committee
   end
 
+  def translate_committee_type(cs_committee)
+    committee_type_code = cs_committee[:committeeType].try(:intern)
+    COMMITTEE_TYPES[committee_type_code].try(:[], :label)
```
Comment (lyttam) [inline app/models/my_committees/committees_module.rb]: @nkutub I separated this into two lines to make it more readable.

**Pull request merged**